src/utils/database/leetcode_question_table.py

```diff
@@ -0,0 +1,61 @@
+"""
+Leetcode Question table module
+"""
+
+from typing import List
+from tinydb import TinyDB, where
+from tinydb.table import Document
+from .db_decorators import validate_insert
+
+class LeetcodeQuestionsTable():
+    """
+    Table to hold all questions pulled from Leetcode
+    """
+    TABLE_LEETCODE_QUESTION_FIELDS = ["id", "title", "title_slug", "difficulty"]
+    TABLE_LEETCODE_QUESTION = "Leetcode_Question"
```

Comment:
move to __init__() as self.table_fields and self.table_name

Reply:
Updated, thanks.